cluster-autoscaler/cloudprovider/aws/aws_wrapper.go

```diff
@@ -18,83 +18,33 @@ package aws
 
 import (
 	"fmt"
-	"sync"
-	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/autoscaling"
-	"k8s.io/apimachinery/pkg/util/clock"
-	"k8s.io/apimachinery/pkg/util/rand"
-	"k8s.io/client-go/tools/cache"
-	klog "k8s.io/klog/v2"
-)
-
-const (
-	launchConfigurationCachedTTL = time.Minute * 20
-	cacheMinTTL                  = 120
-	cacheMaxTTL                  = 600
+	"github.com/aws/aws-sdk-go/service/ec2"
 )
 
 // autoScaling is the interface represents a specific aspect of the auto-scaling service provided by AWS SDK for use in CA
-type autoScaling interface {
+type autoScalingI interface {
```

Comment:
maybe rename this to `autoScaler` to be more in line with interface type names?

adding an `I` makes sense to me, but also can be missed easily.

Reply:
So originally we had an `autoScaling` interface and an `ec2I` interface, because without the `I` the `ec2` interface name conflicted with the AWS SDK service (see [here](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/ec2.go#L26) on the master branch).  I added the `I` here to make the two different interfaces consistent, but maybe there's a better solution?  I'll think about it some.

Reply:
makes sense, i was just trying to think about verbs that describe the action for the interfaces. but, as always, naming is hard ;)